doc/manual/command-ref/conf-file.xml

```diff
@@ -29,13 +29,36 @@
     (i.e. <filename>/etc/nix/nix.conf</filename> on most systems), or
     <filename>$NIX_CONF_DIR/nix.conf</filename> if
     <envar>NIX_CONF_DIR</envar> is set.</para>
+
+    <para>Values loaded in that file are not forwarded to the nix-daemon. The
+    client assumes that the daemon has already loaded them.
+    </para>
+  </listitem>
+
+  <listitem>
+    <para>
+      For each folder in <envar>XDG_CONFIG_DIRS</envar>, separated by
+      <literal>:</literal>.
+
+      Load <filename><replaceable>folder</replaceable>/nix/nix.conf</filename>.
+    </para>
   </listitem>
 
   <listitem>
-    <para>The user configuration file
-    <filename>$XDG_CONFIG_HOME/nix/nix.conf</filename>, or
-    <filename>~/.config/nix/nix.conf</filename> if
-    <envar>XDG_CONFIG_HOME</envar> is not set.</para>
+    <para>
+      The user configuration file in
+      <filename>$XDG_CONFIG_HOME/nix/nix.conf</filename> where
+      <envar>XDG_CONFIG_HOME</envar> defaults to
+      <filename>$HOME/.config</filename>.
+    </para>
+  </listitem>
+
+  <listitem>
+    <para>
+      And finally, every files pointed to by
+      <envar>NIX_EXTRA_CONF_FILES</envar>, separated by <literal>:</literal>
```

Comment:
Is this implemented?

Reply:
yes? You can see it in action in the admittedly minimal test.

Reply:
It only uses `NIX_CONF_FILES`, not `NIX_EXTRA_CONF_FILES`.

Reply:
oh ok. This is a typo. Fixed in 976ead5229d776a9559629b279bd7a27f9945111